### ISSUE TITLE
[Enhancement]: ECS Fargate support for fargateTaskRetirementWaitPeriod

### ISSUE THREAD

**Issue by piotr-lwks** [hashicorp/terraform-provider-aws#33583]
### Description

A recent update allows configuring ECS "task retirement wait period" to either 0, 7 or 14 days with:
Fargate waits for a period of time before starting automatic task retirements after a security update.
This affects ECS Services with Fargate setup but standalone tasks are unaffected (and responsibility of aws customer).

> aws ecs put-account-setting-default --name fargateTaskRetirementWaitPeriod --value 7

### Affected Resource(s) and/or Data Source(s)

* aws_ecs_account_setting_default
* 

### Potential Terraform Configuration

```terraform
resource "aws_ecs_account_setting_default" "defaults" {
  name  = "taskLongArnFormat"
  value = "enabled"

  fargate_retirement_wait_period = 14
}
```

### References

* https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-account-settings.html#fargate-retirement-wait-time

### Would you like to implement a fix?

Yes

**Comment by trevorrea** [comment]
Duplicate of https://github.com/hashicorp/terraform-provider-aws/issues/33422

**Comment by piotr-lwks** [comment]
True. No idea why I did not find it. Closing
